docs/concepts/decentralized-storage/decentralized-identifiers.md

```diff
@@ -8,7 +8,7 @@ labels:
 ---
 # Decentralized Identifiers
 
-_(Requires the [DID amendment][])_
+{% amendment-disclaimer name="DID" /%}
```

Comment:
The location of the amendment-disclaimer is not consistent across concepts topics. In some cases, such as this and AMM concept pages, the disclaimer is the first thing after the page title. But on pages such as Price Oracles and Deep Freeze, the disclaimer is after the introductory paragraph(s). Can we make it consistent so that the disclaimers are shown after the page title (or section heading) when they apply to the entire page (or section)?

Reply:
Will move the others to the bottom of the relevant section (as noted, bottom is preferable so that the disclaimer doesn't get treated as the summary of the page)